ddl/views/artist_coin_prices.sql

```diff
@@ -30,15 +30,19 @@ CREATE VIEW artist_coin_prices AS
         damm_v2.price AS damm_v2_price,
         dbc.price AS dbc_price,
         stats.price AS stats_price,
+        pools.price_usd AS pools_price_usd,
         COALESCE(
             damm_v2.price,
             dbc.price,
-            stats.price
+            stats.price,
+            pools.price_usd
         ) AS price
     FROM artist_coins
     LEFT JOIN dbc ON artist_coins.mint = dbc.mint
     LEFT JOIN damm_v2 ON artist_coins.mint = damm_v2.mint
-    JOIN artist_coin_stats AS stats 
-        ON stats.mint = artist_coins.mint;
+    LEFT JOIN artist_coin_stats AS stats 
+        ON stats.mint = artist_coins.mint
+    LEFT JOIN artist_coin_pools AS pools
```

Comment:
this table is effectively deprecated, I think we can leave this out. was planning to drop this table entirely at some point